framework/src/wifi_manager/wifi_manager_cb.c

```diff
@@ -110,40 +110,58 @@ void _handle_user_cb(_wifimgr_usr_cb_type_e evt, void *arg)
 		switch (evt) {
 		case CB_STA_CONNECTED:
 			WM_LOG_INFO("[WM] call sta connect success event\n");
-			cbk->sta_connected(WIFI_MANAGER_SUCCESS);
+			if (cbk->sta_connected) {
```

Comment:
How about adding logs for error when callback function is NULL?

Reply:
Some applications can use wifi manager not to register all callback functions. so it isn't error even though all callbacks aren't registered.